go.work

```diff
@@ -0,0 +1,10 @@
+go 1.19
```

Comment:
how does this work? go.work instead of go.mod?

Reply:
Think this is part of go workspaces

Reply:
`go.work` is a golang workspace. It's there OOTB monorepo management so we can co-locate our dependencies and have them interoperate in the same git repository.

This helps ensure that these dependencies can be versioned and released together.

https://go.dev/doc/tutorial/workspaces

Reply:
There are a few other things we need to do to ensure all dependencies are properly installed external to the go workspace (once this workspace is versioned).